Model/GetInventoryRequestFromOrder.php

```diff
@@ -109,7 +109,7 @@ private function getAddressFromOrder(OrderInterface $order): ?AddressInterface
     {
         /** @var Address $shippingAddress */
         $shippingAddress = $order->getShippingAddress();
-        if ($shippingAddress === null) {
+        if ($shippingAddress == null) {
```

Comment:
Umm, I don't think the PHPStan error should be resolved like this, maybe look into a better solution? We're basically opening up the comparator here and making it less precise, maybe we need to check some other value?